### DISCUSSION TITLE
README.md -> README.adoc?

### DISCUSSION THREAD

**Discussion by polyglot-jones** [tajmone/ST4-Asciidoctor#28] [General]
@tajmone Is there any reason we shouldn't convert README.md to AsciiDoc? I assume it was started back in the day when GitHub didn't support adoc, and no one ever changed it, yes?

**Comment by tajmone** [comment]
The reason is because Package Control expects all README files to be in GitHub Flavored Markdown, otherwise it can't render them when the package is submitted to the official channel. I know, it sucks, but at least the main README has to be in MD, all other docs can be in any other format supported by GH, AsciiDoc being the obvious choice for a package like this.
